### DISCUSSION TITLE
FORM / Issues ID

### DISCUSSION THREAD

**Discussion by DuPlooydeLange** [iTwin/itwinjs-core#1711] [Q&A] [answered]
Fairly new to this.

Trying to get and post data to forms and issues using the review API.
But for some reason I cannot get the required ID correct.

What is the exact requirement for the ID as I am using the form ID but its not working

Thanks in Advance

**Comment by DuPlooydeLange** [comment]
![FormID](https://user-images.githubusercontent.com/86424480/123291934-6e5b7500-d4c7-11eb-95dd-cff4e14c3525.png)

**Reply by mgooding** [reply]
I've pinged the appropriate developer on our team - you should hear something shortly.

**Reply by DuPlooydeLange** [reply]
Thanks and much appreciated

**Comment by jfisher-bentley** [comment, marked as answer]
You have to do everything from within the API, you will not be able to grab the id from our URL and use that in the API.
You need to get the issues for the project:
https://developer.bentley.com/api-groups/project-delivery/apis/issues-v1/operations/get-project-issues/
Then those ids can be used to get a specific issue's details:
https://developer.bentley.com/api-groups/project-delivery/apis/issues-v1/operations/get-issue-details/

**Reply by DuPlooydeLange** [reply]
Thanks that helped
